minishift-ci-index.yaml

```diff
@@ -75,3 +75,8 @@
           ci_project: '{name}'
           ci_cmd: '/bin/bash centos_ci.sh'
           timeout: '20m'
+      - '{git_repo}':
```

Comment:
+1 Looks good to me. Maybe we can as part of this pull request add Lala and Preveen to the admin list.